nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -207,20 +309,75 @@ Nav2Panel::Nav2Panel(QWidget * parent)
   paused_->assignProperty(navigation_mode_button_, "toolTip", resume_msg);
   paused_->assignProperty(navigation_mode_button_, "enabled", true);
 
+  paused_->assignProperty(save_waypoints_button_, "text", "Save WPs");
+  paused_->assignProperty(save_waypoints_button_, "enabled", false);
+
+  paused_->assignProperty(load_waypoints_button_, "text", "Load WPs");
+  paused_->assignProperty(load_waypoints_button_, "enabled", false);
+
+  paused_->assignProperty(pause_waypoint_button_, "text", "Pause WP");
+  paused_->assignProperty(pause_waypoint_button_, "enabled", false);
+
+  paused_->assignProperty(store_initial_pose_checkbox_, "enabled", false);
```

Comment:
Should the paused state have an explicit policy for the number of loops text field?

Reply:
Yes, I guess that's just the same confusion as the above..

Reply:
I don't see any policy for `nr_of_loops_` under `paused_`

Reply:
Ah! Got what you meant! 

The logic is somewhere here:

https://github.com/ros-planning/navigation2/blob/3d612b2af42daebdc6099541acde85124c0b3c50/nav2_rviz_plugins/src/nav2_panel.cpp#L651-L656

Reply:
I get your point, 

seems that it is taken care here:

https://github.com/ros-planning/navigation2/blob/3d612b2af42daebdc6099541acde85124c0b3c50/nav2_rviz_plugins/src/nav2_panel.cpp#L235

when you go into the `accumulated_wp_` state it is already disabled. Since pausing comes after that, the `nr_of_loops_` never gets enabled.

Reply:
I know, but being explicit about all of the states of all the buttons in all of the states in the machine is useful to make sure no later things create odd conditions